clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -811,7 +812,8 @@ private void handleCompletedReceives(List<ClientResponse> responses, long now) {
                     req.header.apiKey(), req.header.correlationId(), responseStruct);
             }
             // If the received response includes a throttle delay, throttle the connection.
-            AbstractResponse body = AbstractResponse.parseResponse(req.header.apiKey(), responseStruct);
+            AbstractResponse body = AbstractResponse.
+                parseResponse(req.header.apiKey(), responseStruct, req.header.apiVersion());
```

Comment:
Is there a coding standard to structure the L815-816? I wonder if we could merge both.

Reply:
The reason for having two lines is to avoid a line which is too long.